src/app/Components/MovieList/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, connect } from 'react-redux';
+
+import { getMovies } from './duck/operations';
+import { SUCCESS } from '../../../common/constants';
+
+const renderMovies = (movies) => (
+  <div className="flex mt-8 ml-24 mr-24 flex-wrap">
+    {movies.map(({ poster_url, title }) => (
+      <div className="mr-1 mb-4" key={title}>
+        <img src={poster_url} alt={title} />
+      </div>
+    ))}
+  </div>
+);
+
+const renderTitle = (title) => (
+  <div className="">
+    <h1 className="ml-24 text-3xl font-semibold">{title}</h1>
+  </div>
+);
+
+const MovieList = (props) => {
+  const dispatch = useDispatch();
+  const [movies, setMovies] = useState([]);
+
+  useEffect(() => {
+    dispatch(getMovies());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    const { state, payload: movies } = props.movieList.status;
+
+    if (state === SUCCESS) {
+      setMovies(movies);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [props.movieList]);
+
+  return (
+    <main>
+      {renderTitle('My List')}
+      {renderMovies(movies)}
+    </main>
+  );
+};
```

Comment:
What do you think of adding something to show the user that the movies are loading? And also a message in case the was an error.